source-test/tsconfig.json

```diff
@@ -5,7 +5,9 @@
     "noImplicitAny": true
   },
   "files": [
-    "File.ts",
+    "file.ts",
```

Comment:
Is there a reason you renamed this file? git sees it as a delete+add rather than a modify